examples/item/item.json

```diff
@@ -34,7 +34,18 @@
       ]
     ]
   },
-  "links": [],
+  "links": [
+    {
+      "rel": "self",
+      "href": "/home/fabian/dev/stactools-packages-datacube/examples/item/item.json",
+      "type": "application/json"
+    },
+    {
+      "rel": "self",
+      "href": "/home/fabian/dev/stactools-packages-datacube/examples/item/item.json",
+      "type": "application/json"
+    }
+  ],
```

Comment:
Duplicated self links, and generally we prefer to _not_ include `self` links in the examples, as they usually are some non-portable path.